src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/updatelistingrequirements/UpdateListingRequirementsHandler.java

```diff
@@ -0,0 +1,210 @@
+package uk.gov.hmcts.reform.sscs.ccd.presubmit.updatelistingrequirements;
+

```

Comment:
It might be worth moving some of those helper methods into a separate utility class.